manifests/07-operator-ibm-cloud-managed.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
```

Comment:
Would you mind explaining how this is going to work now? Seems like we have two distinct deployments now in `manifests` directory. Isn't CVO going to create two deployments? Please clarify.